lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1893,6 +1895,11 @@ bool PlatformAvailability::isPlatformRelevant(StringRef name) const {
   case PlatformKind::iOSApplicationExtension:
     return name == "ios" || name == "ios_app_extension";
 
+  case PlatformKind::macCatalyst:
+  case PlatformKind::macCatalystApplicationExtension:
+    // ClangImporter does not yet support macCatalyst.
+    return false;
```

Comment:
Should these at least have `name == "ios"`, or is that not even worth it until more complete clang support lands?